server/chat-plugins/helptickets.ts

```diff
@@ -619,6 +620,15 @@ const ticketPages: {[k: string]: string} = {
 	confirmroomhelp: `Call a Global Staff member to help`,
 	confirmother: `Call a Global Staff member`,
 };
+const cheatingScenarios = [
+	[`My opponent's Pokemon used moves it couldn't learn`, `It was probably a disguised Zoroark (<psicon pokemon = "zoroark">), which has the ability <a href = "https://dex.pokemonshowdown.com/abilities/illusion">Illusion</a>. This happens often in Random Battles!`],
+	[`My opponent got very lucky (critical hits, freezes, flinches, etc.)`, `Sometimes, <a href = "https://pokemonshowdown.com/pages/rng">that's just how RNG works!</a>`],
+	[`My opponent used six of the same Pokemon / too many Legendaries`, `Certain tiers, like Anything Goes, do not have Species Clause, which normally restricts a player to only one of each Pokemon. In addition, many tiers allow lots of legendaries, and you are allowed to use them!`],
+	[`My Pokemon used a move I didn't choose`, `You accidentally selected the wrong move and didn't notice. It happens more often than you might think!`],
+	[`My Pokemon moved last when it should have moved first`, `You probably accidentally chose a move with negative priority, like Trick Room, Dragon Tail, or Roar.`],
```

Comment:
```suggestion
	[`My Pokemon moved last when it shouldn't have`, `You probably accidentally chose a move with negative priority, like Trick Room, Dragon Tail, or Roar.`],
```
More ambiguous phrasing for FFA/Multi battles, also maybe mentioning the opponent's side might have Tailwind up?

Reply:
Great catch, I like that a lot better!